firebase_admin/project_management.py

```diff
@@ -147,46 +324,130 @@ def __init__(self, name, app_id, display_name, project_id):
 
     @property
     def name(self):
-        """The fully qualified resource name of this Android App."""
+        """The fully qualified resource name of this Android or iOS app."""
         return self._name
 
     @property
     def app_id(self):
-        """The globally unique, Firebase-assigned identifier of this Android App.
+        """The globally unique, Firebase-assigned identifier of this Android or iOS app.
 
-        This ID is unique even across Apps of different platforms, such as iOS Apps.
+        This ID is unique even across apps of different platforms.
         """
         return self._app_id
 
     @property
     def display_name(self):
-        """The user-assigned display name of this Android App."""
+        """The user-assigned display name of this Android or iOS app."""
         return self._display_name
 
     @property
     def project_id(self):
-        """The permanent, globally unique, user-assigned ID of the parent Firebase Project."""
+        """The permanent, globally unique, user-assigned ID of the parent Firebase project."""
         return self._project_id
 
 
 class AndroidAppMetadata(_AppMetadata):
-    """Android-specific information about an Android Firebase App."""
+    """Android-specific information about an Android Firebase app."""
 
-    def __init__(self, name, app_id, display_name, project_id, package_name):
+    def __init__(self, package_name, name, app_id, display_name, project_id):
```

Comment:
It seems to me the argument order you had earlier is the more appropriate one. Commons args followed by more type specific args.

Reply:
I actually wanted to leave it as the last argument as well, but then I will have trouble in `_get_app_metadata` in line 498 because there the parameter name would be different between iOS and Android, forcing me to pass it as the first positional argument and the rest as keyword arguments (which must follow positional arguments). On the other hand, I can remove keywords from all the other arguments. Please let me know if you have any suggestions.